modules/hydrodyn/src/Morison.f90

```diff
@@ -3313,8 +3421,15 @@ SUBROUTINE Morison_CalcOutput( Time, u, p, x, xd, z, OtherState, y, m, errStat,
             CALL MrsnOut_WriteOutputs( p%UnOutFile, Time, y, p, errStat, errMsg )         
          END IF
       END IF
-      
-   
+
+
+      ! map the motion to the visulization mesh
+      if (p%VisMeshes) then
+         !FIXME: error handling is incorrect here (overwrites all previous errors/warnings)
```

Comment:
Should this be fixed before merging?

Reply:
There's a significant amount of reworking of the error handling in Morison required to really address this.  Since there are more HydroDyn changes in the `dev` and `dev-unstable` branches, I would rather wait and deal with error handling there instead of introduce potential merge conflicts now.